.github/CODEOWNERS

```diff
@@ -351,6 +351,21 @@
 /x-pack/test/case_api_integration @elastic/security-threat-hunting
 /x-pack/plugins/lists @elastic/security-detections-response
 
+## Security Solution sub teams - security-onboarding-and-lifecycle-mgt 
+/x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps @elastic/security-onboarding-and-lifecycle-mgt
```

Comment:
I would suggest we group and sort these paths by folder. `.../security_solution/server/...`, `.../security_solution/public/...` etc

Reply:
Lines 361-362 can move below line 356.
Also, line 357 can move right above line 359. 
So that all `/x-pack/plugins/security_solution/server`, `/x-pack/plugins/security_solution/public` and `/x-pack/plugins/security_solution/common` paths are grouped together.